py3status/modules/tor_rate.py

```diff
@@ -15,6 +15,7 @@
         (default "↑ {up} ↓ {down}")
     format_value: A string describing how to format the transfer rates
         (default "[\?min_length=12 {rate:.1f} {unit}]")
+    hide_if_not_running: Hide socket error if not running (default False)
```

Comment:
I'd say "Hide module if tor is not running locally"

Reply:
I write this because finally, with this, i can assume to have my local or remote tor server not joignable and so hide the error.
Btw, we got the others errors if the server is reachable but return bad authent or another error.

Reply:
```    
hide_socket_errors: Hide errors connecting to Tor control socket
        (default False)
```
(?)

Reply:
I don't have a preference. `hide socket errors` or `hide module when local tor is not running` is fine, but we think we shouldn't have to tell users what to toggle. Some could want it off anyway to ensure that they do run local tor.

I think `hide_if_not_running` is a bit of poor config name to use so I suggested `hide_socket_errors` rather than `hide_errors` -- which probably would hide all errors. Do you have a suggestion for a config name that describes `hide module if local tor is running` ?

Reply:
how about `hide_if_no_tor_process`? We could actually change the implementation a bit, instead of doing the check inside `except SocketError` we could check the existence of the `tor` process in the very beginning of the `tor_rate` function.

Reply:
Yeah you are right